x/group/server/invariants.go

```diff
@@ -169,3 +181,129 @@ func groupTotalWeightInvariant(ctx sdk.Context, groupTable orm.Table, groupMembe
 	}
 	return msg, broken, err
 }
+
+func tallyVotesSumInvariant(ctx sdk.Context, proposalTable orm.AutoUInt64Table, groupMemberTable orm.PrimaryKeyTable, voteByProposalIndex orm.UInt64Index, groupAccountTable orm.PrimaryKeyTable) (string, bool, error) {
+	var msg string
+	var broken bool
+
+	var proposal group.Proposal
+	var groupAcc group.GroupAccountInfo
+	var groupMem group.GroupMember
+	var vote group.Vote
+
+	proposalIt, err := proposalTable.PrefixScan(ctx, 1, math.MaxUint64)
+	if err != nil {
+		return msg, broken, err
+	}
+	defer proposalIt.Close()
+
+	for {
+
+		totalVotingWeight := apd.New(0, 0)
+		yesVoteWeight := apd.New(0, 0)
+		noVoteWeight := apd.New(0, 0)
+		abstainVoteWeight := apd.New(0, 0)
+		vetoVoteWeight := apd.New(0, 0)
+
+		_, err := proposalIt.LoadNext(&proposal)
+		if orm.ErrIteratorDone.Is(err) {
+			break
+		}
+
+		address, err := sdk.AccAddressFromBech32(proposal.Address)
+		if err != nil {
+			return msg, broken, err
+		}
+
+		err = groupAccountTable.GetOne(ctx, address.Bytes(), &groupAcc)
+		if err != nil {
+			break
```

Comment:
```suggestion
			return msg, broken, err
```

Reply:
@blushi I don't think we should be using `return msg, broken, err`, assuming no `GroupAccountInfo` with proposal address, we'll be returning an error which panics on running invariants

Reply:
how is it different to the other cases where we do return `msg, broken, err`?

Reply:
Here, we don't even need to go for other cases since there'll be no such `GroupAccountInfo`

Reply:
well if we couldn't find any group account, then that means that we can't verify invariant with the current proposal which belongs to this group account, so for this reason I believe this should error

Reply:
maybe you were right, but `FullAppSimulation` fails because it panics here